hummingbot/connector/gateway/clob_spot/data_sources/xrpl/xrpl_api_data_source.py

```diff
@@ -80,6 +84,37 @@ async def start(self):
     async def stop(self):
         await super().stop()
 
+    async def _update_markets(self):
+        if self._is_loading_markets:
```

Comment:
is better to use asyncio.Event() to manage this, I think that the logic can be improved here.

Reply:
Thanks, replaced that with asyncio.Event() 👍